kale/tests/unit_tests/test_metadata_utils.py

```diff
@@ -37,23 +38,64 @@
      }])
 ])
 def test_validate_volumes_metadata(volumes, target):
-    """Tests that volumes are correctly converted from list into dict
+    """Tests that volumes are correctly converted from list into dict."""
```

Comment:
In this test, the conversion of volume **annotations** from list to dict is tested.
But the general purpose is to test the `_validate_volumes_metadata()` function. So I'd propose the doc string to be
```
Tests volumes validation method
```
or similar.